setup.py

```diff
@@ -50,5 +50,6 @@
           'Programming Language :: Python :: 3.3',
           'Programming Language :: Python :: 3.4',
           'Programming Language :: Python :: 3.5',
+          'Programming Language :: Python :: 3.6',
```

Comment:
It looks like maybe you merged instead of rebased? It's not a problem, I should be able to remove these commits before merging into master.